src/effects/VST/VSTEffect.h

```diff
@@ -288,6 +288,38 @@ struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
    int mProcessLevel{ 1 };  // in GUI thread
 
    VSTEffect* mMaster;     // non-NULL if a slave
+
+#if defined(__WXMAC__)
```

Comment:
Please do an interactive rebase, squashing this commit onto the second one, so that all commits are buildable.

Reply:
ok, now done and force-pushed

Reply:
oops I should delete the "build fix for OSx" commit - it's only whitespaces now